pyproject.toml

```diff
@@ -58,8 +58,8 @@ dependencies = [
     "pyqt6",
     "requests",
     "resfo",
-    "ropt-dakota>=0.22",
-    "ropt[pandas]>=0.22",
+    "ropt-dakota>=0.22,<0.24",
+    "ropt[pandas]>=0.22,<0.24",
```

Comment:
I would make it < 0.23, not 0.24. ropt-0.23 does not exist (it was yanked), but still.

Reply:
Fixed